From 4fde02e923cc6f62a000caa66c12705ae5d2c6d3 Mon Sep 17 00:00:00 2001
From: Oscar Lesta <oscar.lesta@gmail.com>
Date: Sun, 1 Jan 2023 07:35:12 -0300
Subject: Change header guard to not clash with the one from Haiku's Errors.h

It was causing some compilation errors that were a bit puzzling.

diff --git a/src/errors.h b/src/errors.h
index 831980d..89834ce 100644
--- a/src/errors.h
+++ b/src/errors.h
@@ -1,5 +1,5 @@
-#ifndef _ERRORS_H
-#define _ERRORS_H
+#ifndef _ERRORS_CRYPTODOME_H
+#define _ERRORS_CRYPTODOME_H
 
 /** Standard errors common to all ciphers **/
 #define ERR_NULL                1
-- 
2.37.3

